### PR TITLE
Feature/location restaurant

### PR DESCRIPTION
**Pull request by emilio-gambino**
This PR enables our application to record occupancies in restaurants.

### CODE REVIEW

File: app/src/main/java/com/github/sdp/ratemyepfl/activity/MainActivity.kt [hunk 1]
```diff
@@ -40,10 +57,66 @@ class MainActivity : AppCompatActivity() {
...
     }
 
     override fun onSupportNavigateUp(): Boolean {
         return navController.navigateUp(appBarConfiguration)
     }
+
+    /**
+     * Starts the Location Service and binds this Activity as a Listener
+     */
+    private fun startLocationService() {
```
Comment (mbouilloud) [inline app/src/main/java/com/github/sdp/ratemyepfl/activity/MainActivity.kt:71]: We can maybe unify this location service with what Nicolas did in the Map activity. A task for next sprint !

File: app/src/main/java/com/github/sdp/ratemyepfl/database/RestaurantRepository.kt [hunk 2]
```diff
@@ -33,4 +27,29 @@ class RestaurantRepository @Inject constructor() : RestaurantRepositoryInterface
     }
 
     override suspend fun getRestaurantById(id: String): Restaurant? = toItem(getById(id))
+
+    override fun incrementOccupancy(id: String) {
```
Comment (mbouilloud) [inline app/src/main/java/com/github/sdp/ratemyepfl/database/RestaurantRepository.kt:31]: Great job for the transactions ! Your code really helped me understand how it works. I used it for overall grade in the review part of the app :).

**Review by jmettler27 — APPROVED** [review]
Nothing to say in particular, great feature!

File: app/src/main/java/com/github/sdp/ratemyepfl/database/RestaurantRepository.kt (hunk 2 above)
Comment (emilio-gambino) [reply app/src/main/java/com/github/sdp/ratemyepfl/database/RestaurantRepository.kt:31]: Glad it helped ! :)

**Pull request merged**